### PR TITLE
[Misc] Add attention mask

### PR DESCRIPTION
**Pull request by wangxiyuan**
Move attention mark from V0 to common place.
- vLLM version: v0.9.2
- vLLM main: https://github.com/vllm-project/vllm/commit/b942c094e3ab905aeb16f4136353f378e17159e8

### CODE REVIEW

File: vllm_ascend/attention/attention_mask.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        max_seq_len = max(seq_lens, default=0)
+        if max_seq_len <= self._seq_len_cached:
+            self._update_attn_cache(max_seq_len, dtype, device)
+            # FIXME: Currently the mask value of chunked-prefill situation and Prefill-Only situation
+            # is not the same. Fix this in the future when kernel is ready.
+            if self.attn_mask_cache.numel(
+            ) > 1 and self.attn_mask_cache[0][1] > 0:
+                attn_mask = self.get_attn_mask(  # type: ignore
+                    max_seq_len, dtype, device)
+                attn_mask *= -10000
```
Comment (Yikun) [inline vllm_ascend/attention/attention_mask.py:71]: See also here https://github.com/vllm-project/vllm-ascend/pull/1678

**Review by Yikun — APPROVED** [review]

File: vllm_ascend/attention/attention_mask.py (hunk 1 above)
Comment (wangxiyuan) [reply vllm_ascend/attention/attention_mask.py:71]: Added comment there
